src/widgets/Catalog/filters/field.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import ArrowDown from './asserts/arrow-down.svg';
+import ArrowUp from './asserts/arrow-up.svg';
+
+type Props = {
```

Comment:
```suggestion
type TFiltersField = {
```
Типы называем с T, желательно давать осознанное название.